packages/backend/src/projectAdapters/dbtBaseProjectAdapter.ts

```diff
@@ -93,7 +98,26 @@ export class DbtBaseProjectAdapter implements ProjectAdapter {
 
         try {
             const fileContents = await fs.readFile(configPath, 'utf8');
-            const config = await loadLightdashProjectConfig(fileContents);
+            const config = await loadLightdashProjectConfig(
+                fileContents,
+                async (lightdashConfig) => {
+                    void this.analytics?.track({
+                        event: 'lightdashconfig.loaded',
+                        properties: {
+                            // projectId: projectUuid,
```

Comment:
dont have a clear way to send "projectId", "userId", or "organizationId" to this method without changing loads